packages/core/src/network.js

```diff
@@ -10,7 +10,7 @@ const ALLOWED_RESOURCES = ['Document', 'Stylesheet', 'Image', 'Media', 'Font', '
 // The Interceptor class creates common handlers for dealing with intercepting asset requests
 // for a given page using various devtools protocol events and commands.
 export class Network {
-  static TIMEOUT = 30000;
+  static TIMEOUT = undefined;
```

Comment:
we have something for `networkIdleTimeout` which can have max value of `750ms` which can be provided by users. 

https://github.com/percy/cli/blob/c12668d70252186f061399f00f4f2aebbcc06d8c/packages/core/src/network.js#L75-L78

Reply:
so we'll need to verify once for heavy pages, is `TIMEOUT` honoured over `networkIdleTimeout`

Reply:
networkIdleTimeout is different, is the event, this TIMEOUT is amount of time we will wait for `networkIdleTimeout` to trigger. 

i.e. by default wait for 30 sec for networkIdleTimeout to hit, which will only be hit if network is idle for 750ms